index.bs

```diff
@@ -604,7 +617,10 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
   <dd><a>Color temperature</a> in use for the white balance calculation of the scene. This field is only significant if {{MediaTrackSettings/whiteBalanceMode}} is {{manual}}.</dd>
 
   <dt><dfn dict-member for="MediaTrackSettings"><code>exposureCompensation</code></dfn></dt>
-  <dd>Current <a>exposure compensation</a> setting.  A value of 0 EV is interpreted as no exposure compensation.</dd>
+  <dd>Current <a>exposure compensation</a> setting.  A value of 0 EV is interpreted as no exposure compensation. This field is only significat if {{MediaTrackSettings/exposureMode}} is </dd>
```

Comment:
s/`significat if {{MediaTrackSettings/exposureMode}} is`/`significant if {{MediaTrackSettings/exposureMode}} is {{MeteringMode/continuous}} or {{MeteringMode/single-shot}}`/